src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/persistence/SQLitePersistor.java

```diff
@@ -362,7 +362,7 @@ private JsonElement readTemperatureProfileRca(String rca) {
         response.addNestedSummaryList(nodeLevelDimSummary);
       }
     } catch (DataAccessException dex) {
-      LOG.error("Failed to read temperature profile RCA for {}", rca, dex);
+      LOG.warn("Failed to read temperature profile RCA for {}. Msg: {}", rca, dex.getMessage());
```

Comment:
This is generally an anti pattern. Is there a reason you're throwing away the stacktrace?

Reply:
Not being able to find the RCA table is justified if the query is made early enough that one is not generated. The long stack trace gives people an impression of an error, when this can happen. That's why I removed the stacktrace with just a message.